### DISCUSSION TITLE
How to refresh Client code after deploying

### DISCUSSION THREAD

**Discussion by tuhin24** [aws-samples/aws-saas-factory-ref-solution-serverless-saas#66] [Q&A] [answered]
Hi, 
I modified create.component.html and create.component.ts to add additional fields from the client application. The pipeline status is successful. When I log in to the application page using my tenant credentials and try to create a new product, I do not see the changes I made on the product page. How do I see the changes I made?

Thanks

**Comment by ujwalbukka** [comment]
Hi @tuhin24 - The code which deploys all client UI applications is in deployment.sh script. Specifically from [line64](https://github.com/aws-samples/aws-saas-factory-ref-solution-serverless-saas/blob/main/deployment.sh#L64). Based on which site you are deploying you can run that client UI code from the deployment.sh script.

**Comment by tuhin24** [comment]
When I upload my client code (create.component.html and create.component.ts ) to CodeCommit, it appears the pipeline is trying to redeploy the changes and shows success. But I do not see my changes when I log in to the UI. Are you suggesting that I have to manually run the script to see my changes?

**Comment by ujwalbukka** [comment, marked as answer]
Hi @tuhin24 - yes, UI changes are only deployed through deployment.sh script. Which you have to manually run it.
